tonic/src/metadata/mod.rs

```diff
@@ -33,6 +33,9 @@ pub use self::value::MetadataValue;
 
 pub(crate) use self::map::GRPC_TIMEOUT_HEADER;
 
+/// HTTP Header `content-type` value for gRPC calls.
+pub const GRPC_CONTENT_TYPE: &str = "application/grpc";
```

Comment:
Could we have this be of type `HeaderValue`?

Reply:
@djc hello! I fixed it here: https://github.com/hyperium/tonic/pull/1648